packages/lumx-react/src/hooks/useDelayedVisibility.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+
+const ANIMATION_DELAY = 400;
```

Comment:
Animation delay is 400 for dialog but only 200 for notifications

Reply:
@matmkian added those changes! thanks for the review!